src/applications/static-pages/wizard/tests/bdd-526.unit.spec.jsx

```diff
@@ -201,8 +202,9 @@ describe('Form 526 Wizard', () => {
     wrapper.unmount();
   });
 
-  // disagreeing flow
-  it('should allow disability form start when filing a new claim', () => {
+  // disagreeing flow - skipping since wizard is depreacted and 526 subtask will
+  // be implemented soon
+  it.skip('should allow disability form start when filing a new claim', () => {
```

Comment:
Note that this entire wizard flow test is already covered by an e2e test - src/applications/disability-benefits/all-claims/tests/all-claims-wizard.cypress.spec.js